src/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests.csproj

```diff
@@ -5,13 +5,18 @@
     <IsTestProject>true</IsTestProject>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>../GSoft.Authentication.ClientCredentialsGrant.snk</AssemblyOriginatorKeyFile>
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
```

Comment:
`Duende.IdentityServer` packages don't are not strong-named but we don't care because this is a test project.

Reply:
Yep I made a mistake in my sentence. And yes we don't care since it is a test project. As long as all third-party dependencies in our production library are strong-signed, we're good (for instance, this is the case for `IdentityModel`).